### PR TITLE
Fix: Issue #1176 App crashing on data connection.

### LINKED ISSUE #1176: App crashing when on data connection.
**Actual Behaviour**

screenshot_20170227-234553

When the snackbar appears, if instead of clicking yes we click on any track then app getting crashed.

screenshot from 2017-02-27 23 44 55

**Expected Behaviour**

App should not crash.

@the-dagger sending the PR. Please review.

### PR DESCRIPTION
**Pull request by harshitagupta30**
Fixes #1176 

Changes: Null pointer check on cursor in `DatabaseOperations.java`.

@the-dagger Please review! :+1:

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java [hunk 1]
```diff
@@ -970,13 +988,14 @@ public Session getSessionbySessionname(String sessionName, SQLiteDatabase mDb) {
         );
 
         ArrayList<Integer> ids = new ArrayList<>();
-        cursor.moveToFirst();
-        while (!cursor.isAfterLast()) {
-            ids.add(cursor.getInt(cursor.getColumnIndex(DbContract.Bookmarks.SESSION_ID)));
-            cursor.moveToNext();
-        }
+        if(cursor.moveToFirst() && cursor != null) {
```
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java]: @harshitagupta30 the null check should come first in the if statement.

**Comment by the-dagger** [comment]
@harshitagupta30 could you fix codacy warnings?
Also do change the default indentation as that makes it hard to see the actual changes.

Please update the PR with these changes.
Thanks.

**Comment by harshitagupta30** [comment]
@the-dagger I'll do that and update the PR! :+1:

**Comment by harshitagupta30** [comment]
@the-dagger `public` access modifier was used everywhere unnecessarily which was giving multiple warnings. So I removed the public. Do I need to change or is it fine?

**Comment by the-dagger** [comment]
@harshitagupta30 I'll suggest changing back the indentation in the project file.
The diff shows many unnecessary changes which makes it hard to see the actual changes.

**Comment by harshitagupta30** [comment]
@the-dagger I am sorry I'll do that!

**Comment by harshitagupta30** [comment]
@the-dagger as I am adding if condition it automatically indent the whole block of code by itself.

**Comment by the-dagger** [comment]
Oh I see.
Can't do anything then.
Thanks

**Comment by harshitagupta30** [comment]
@the-dagger Just a quick suggestion! Instead of viewing here you can run this branch/copy this file compare it with development brach in android studio which will be a hell lot easier as compared to here! ;) :smile:  

Sorry for the trouble!

**Comment by the-dagger** [comment]
Yes, that's what I will be doing. :p
I'll let you know after I've reviewed it.

**Comment by mananwason** [comment]
LGTM merging

**Pull request merged**